pkg/command/context.go

```diff
@@ -1281,28 +1286,28 @@ func newUpdateCtxCmd() *cobra.Command {
 		Short:  "Update an aspect of a context (subject to change)",
 		Hidden: true,
 	}
-	taeActiveResourceCmd.Flags().StringVarP(&projectStr, "project", "", "", "project name to be set as active")
-	taeActiveResourceCmd.Flags().StringVarP(&spaceStr, "space", "", "", "space name to be set as active")
+	tanzuActiveResourceCmd.Flags().StringVarP(&projectStr, "project", "", "", "project name to be set as active")
+	tanzuActiveResourceCmd.Flags().StringVarP(&spaceStr, "space", "", "", "space name to be set as active")
 
 	updateCtxCmd.AddCommand(
-		taeActiveResourceCmd,
+		tanzuActiveResourceCmd,
 	)
 	return updateCtxCmd
 }
 
-// taeActiveResourceCmd updates the TAE(Tanzu Application Engine) active resource referenced by tae context
+// tanzuActiveResourceCmd updates the tanzu active resource referenced by tanzu context
 //
 // NOTE!!: This command is EXPERIMENTAL and subject to change in future
-var taeActiveResourceCmd = &cobra.Command{
-	Use:               "tae-active-resource CONTEXT_NAME",
-	Short:             "updates the Tanzu Application Engine(TAE) active resource for the given TAE context (subject to change).",
+var tanzuActiveResourceCmd = &cobra.Command{
+	Use:               "tanzu-active-resource CONTEXT_NAME",
+	Short:             "updates the tanzu active resource for the given context of type tanzu (subject to change).",
```

Comment:
Could you add an uppercase letter at the start of the short help?  All others start with an uppercase

Reply:
Updated. Thanks!